src/core/lib/ConsiderationEncoder.sol

```diff
@@ -571,8 +574,17 @@ contract ConsiderationEncoder {
         uint256 size,
         uint256 memoryLocationForOrderHashes
     ) {
-        // Get free memory pointer to write calldata to.
-        ptr = getFreeMemoryPointer();
+        // Derive offset to pre `OrderFulfilled`'s spent item event data
+        // using base offset & total original recipients.
+        ptr = MemoryPointer.wrap(
```

Comment:
this could be unchecked; if totalOriginalAdditionalRecipients is big enough to overflow the addition or zero out the shift, then it'd fail the hash derivation that goes down prior to reaching here

Reply:
fixed it in the next commit.